app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,8 @@
     <!-- Toasts -->
     <string name="toast_recording_start">Recording started</string>
     <string name="toast_recording_finish">Recording saved to</string>
+    <string name="toast_recording_paused">Recording paused</string>
+    <string name="toast_recording_resumed">Recording resumed</string>
```

Comment:
The string `toast_recording_resumed` is not used anywhere. Did you forget to add a toast?